src/main/java/com/amazon/inspector/jenkins/amazoninspectorbuildstep/sbomgen/SbomgenDownloader.java

```diff
@@ -44,12 +49,12 @@ private static FilePath downloadFile(String url, FilePath workspace) throws IOEx
         return sbomgenZip;
     }
 
-
-    @SuppressFBWarnings()
-    private static String unzipFile(FilePath zip) throws IOException, InterruptedException, ExecutionException {
-        FilePath destination = zip.getParent().child(zip.getRemote().replace(".zip", ""));
-        Future<String> callable = zip.actAsync(new DownloaderCallable(destination.getRemote()));
-
-        return callable.get();
```

Comment:
If possible, continue to use this unzip function + downloader callable, it was added to fix a customer issue. Some customers reported the plugin doing filesystem operations on the host machine instead of the agent machine.